data/console/test_java.sh

```diff
@@ -298,7 +307,8 @@ for java_version in $(cat $LIST_ALL_INSTALLED_VERSIONS); do
     # Delete the successfully compiled files: class and binary of Hello World
     rm $HELLO.class
     echo
-done
 
 # Breakdown
-clean_up;
+if [[ "$1" != "--transactional-server" ]]; then
```

Comment:
This seems to be necessary on transactional systems as well

Reply:
checked updated now.